components/script/dom/range.rs

```diff
@@ -583,8 +583,9 @@ impl RangeMethods for Range {
         }
 
         // Steps 5-12.
-        let (first_contained_child, last_contained_child, contained_children) =
-            self.contained_children()?;
+        let first_contained_child = self.contained_children()?.first_partially_contained_child;
```

Comment:
Rather than calling this method three times, we can do:
```
let ContainedChildren {
  first_partially_contained_child,
  last_partially_contained_child,
  contained_children,
} = self.contained_children()?;

Reply:
hmm, makes sense

Reply:
but for destructuring them in the next step, we'll have to do:
```
let ContainedChildren {
    first_partially_contained_child: first_contained_child,
    last_partially_contained_child: last_contained_child,
    contained_children,
} = self.contained_children()?;
```